ansible/roles/ecryptfs/tasks/main.yml

```diff
@@ -74,29 +74,41 @@
     - after-reboot
 
 - name: Mount data drive
-  sudo: yes
+  become: true
   shell: "mount -t ecryptfs -o key=passphrase:passphrase_passwd={{ ECRYPTFS_PASSWORD }},user,noauto,ecryptfs_cipher=aes,ecryptfs_key_bytes=32,ecryptfs_unlink_sigs,ecryptfs_enable_filename_crypto=y,ecryptfs_fnek_sig={{ password_hash.stdout }},verbosity=0 {{ encrypted_root }}/ {{ encrypted_root }}/"
   when: encrypted_root not in mtab_contents.stdout
   tags:
     - mount-ecryptfs
     - after-reboot
 
 - name: "Create {{ encrypted_root }}/blobdb directory for setups that do not use NFS"
-  sudo: yes
+  become: true
   when: not shared_drive_enabled
   file: path="{{ encrypted_root }}/blobdb" owner=nobody group="{{ shared_dir_gid }}" mode=0775 state=directory
   tags:
     - ecryptfs-blobdb
 
+- name: Create shared data dir
```

Comment:
Is there a way to reuse the `Create shared data dir` task that's in ansible/roles/shared_dir/tasks/setup_host.yml? ... Or if it's not possible, maybe add a comment to point to each other, so that if we change this task, we know to change it in both places.

Reply:
good call, let me see what I can do.